composer.json

```diff
@@ -16,7 +16,7 @@
     "homepage": "http://sabre.io/event/",
     "license": "BSD-3-Clause",
     "require": {
-        "php": "^7.1|^8.0"
+        "php": "^7.1 || ^8.0"
```

Comment:
Note: I adjusted this to have the same `|||` or format as in other repos, just to be consistent.